assets/style.css

```diff
@@ -0,0 +1,149 @@
+:root {
+	--bs-body-bg: #ebebeb;
+}
+
+.navbar {
+	padding: 20px 40px 20px 40px;
+}
+
+.search-field {
+	position: relative;
+}
+
+.search-icon {
+	position: absolute;
+	padding: 10px;
+	pointer-events: none;
+	left: 0px;
+}
+
+.search-input {
+	padding-left: 30px;
+}
+
+.banner {
+	padding-top: 4rem;
+	padding-bottom: 4rem;
+}
+
+.nav-pills {
+	--bs-nav-pills-border-radius: 0.375rem;
+	--bs-nav-pills-link-active-color: var(--bs-black);
+	--bs-nav-pills-link-active-bg: var(--bs-gray-300);
+}
+
+.nav {
+	--bs-nav-link-color: var(--bs-gray-700);
+	--bs-nav-link-hover-color: var(--bs-black);
+	--bs-nav-link-disabled-color: #6c757d;
+}
+
+.nav-list {
+	background-color: white;
+	border-radius: 0.375rem;
+}
+
+.tab-content {
+	min-height: auto;
+	background-color: white;
+	border-radius: 0.375rem;
+}
+
+.anchor {
+	position: relative;
+	top: -100px;
+	display: block;
+	height: 0;
+	overflow: hidden;
+}
+
+.link-card {
+	list-style: none;
+	display: flex;
+	border: 1px solid hsla(240, 6%, 83%, 0.63);
+	border-radius: 0.5rem;
+	min-height: 200px;
+	position: relative;
+	line-height: 1.4;
+	padding: 1em 1.3em;
+	border-radius: 0.35rem;
+	color: var(--text-color);
+	background-image: linear-gradient(120deg, #fff 45%, #f5f5f5);
+	display: block;
+}
+
+.link-card:is(:hover, :focus-within) {
+	filter: drop-shadow(0 10px 10px rgba(76, 76, 109, 0.07059));
+}
+
+/* h5 {
+  margin: 0;
+  transition: color 0.6s cubic-bezier(0.22, 1, 0.36, 1);
+}
+
+p {
+  margin-top: 0.75rem;
+  margin-bottom: 0;
+} */
```

Comment:
Consider leaving a comment about why this is commented out (or removing it).

Reply:
removed :)